rpm/centos-8/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG GO_IMAGE
+ARG DISTRO=centos
+ARG SUITE=8
+ARG BUILD_IMAGE=${DISTRO}:${SUITE}
+
+FROM ${GO_IMAGE} AS golang
+
+FROM ${BUILD_IMAGE}
+ENV GOPROXY=direct
+ENV GO111MODULE=off
+ENV GOPATH=/go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+ENV AUTO_GOPATH 1
+ENV DOCKER_BUILDTAGS seccomp selinux
+ENV RUNC_BUILDTAGS seccomp selinux
+ARG DISTRO
+ARG SUITE
+ENV DISTRO=${DISTRO}
+ENV SUITE=${SUITE}
+RUN yum install -y rpm-build rpmlint
+COPY SPECS /root/rpmbuild/SPECS
+# Overwrite repo that was failing on aarch64
+RUN sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Sources.repo
```

Comment:
Looks like this is no longer needed for CentOS 8 (it was added in https://github.com/docker/docker-ce-packaging/commit/0fd20b90cd322f0eeb06440833ba119b26f44d05 / https://github.com/docker/docker-ce-packaging/pull/142):

The `centos:7` (`linux/arm64`) image indeed has this repo:

```bash
docker run --rm --platform=linux/arm64 centos:7 sh -c 'cat /etc/yum.repos.d/CentOS-Sources.repo | grep altarch'
baseurl=http://vault.centos.org/altarch/$releasever/extras/Source/
baseurl=http://vault.centos.org/altarch/$releasever/centosplus/Source/
```

But they are no longer in the `centos:8` image:

```bash
docker run --rm --platform=linux/arm64 centos:8 sh -c 'cat /etc/yum.repos.d/CentOS-Sources.repo | grep altarch'
```

Reply:
@thaJeztah I don't see it in `centos:7` either
```
🐳 ~/go-workspace/src/github.com/docker/docker-ce-packaging$ docker run --rm --platform=linux/arm64 centos:7 sh -c 'cat /etc/yum.repos.d/CentOS-Sources.repo | grep altarch'
🐳 ~/go-workspace/src/github.com/docker/docker-ce-packaging$ docker run --rm --platform=linux/arm64 centos:8 sh -c 'cat /etc/yum.repos.d/CentOS-Sources.repo | grep altarch'
```